include/ftl/atomic_counter.h

```diff
@@ -158,6 +158,30 @@ class AtomicCounter {
 		return prev;
 	}
 
+	/**
+	 * A wrapper over std::atomic_uint::compare_exchange_strong()
+	 *
+	 * The compare_exchange_strong *with* be atomic, but this function as a while is *not* atomic
```

Comment:
Pedantic, but *with* should be *will*

Reply:
Also *while* should be *whole*

Reply:
Haha good catch, my brain isn't in gear today :smile: